webapp/app.py

```diff
@@ -4,19 +4,13 @@
 The web frontend for the snap store.
 """
 
-import flask
-from werkzeug.contrib.fixers import ProxyFix
-from werkzeug.debug import DebuggedApplication
+from canonicalwebteam.flask_base.app import FlaskBase
 
 import talisker.flask
```

Comment:
Sorry I didn't catch this in the previous review round, but these 2 lines should also be removed as they're in flask-base:

https://github.com/canonical-web-and-design/canonicalwebteam.flask-base/blob/master/canonicalwebteam/flask_base/app.py#L85